tests/Integration/APITest.php

```diff
@@ -476,6 +476,103 @@ public function test_deleteContainerTag_shouldFailWhenContainerNotExists()
         $this->api->deleteContainerTag($this->idSite, 9999, $this->idContainerDraftVersion, $idTag = 999);
     }
 
+    public function test_pauseContainerTag_shouldFailWhenNotHavingViewPermissions()
```

Comment:
Mislabelled maybe? Should be WhenNotHavingWritePermissions instead of ViewPermissions?

Reply:
Updated :+1: